[Data] Update user docs for default column names (#52617)
ray-project/ray: doc/source/data/loading-data.rst

            )

        You can also create a :class:`~ray.data.dataset.Dataset` from a list of regular

>         Python objects. In the schema, the default column name will be "item".

Reviewer: Nit: Here and elsewhere:
```suggestion
        Python objects. In the schema, the column name defaults to "item".
```

We adhere to the Google technical writing style guide, which recommends active voice and present tense

    .. tab-item:: Int Range

        To create a synthetic :class:`~ray.data.Dataset` from a range of integers, call
        :func:`~ray.data.range`. Ray Data stores the integer range in a single column called

>         'id'

Reviewer 2: Be sure to have proper punctuation
